bitrise.yml

```diff
@@ -1,243 +1,20 @@
-format_version: "8"
+format_version: "11"
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
 workflows:
```

Comment:
Can we add a `sample` workflow?

Reply:
I tried adding a sample workflow, but then realized that it's not practical for this step because:

- Users need to create a .bitrise.secrets.yml with a lot of secret env vars. Most of the time it's easier to test in CI, but if someone really needs local debugging then they can run one of the E2E workflows.
- A successful deployment relies on an incrementing `versionCode`, a new release needs to have a higher `versionCode` than the previous version. We achieve this by using `$BITRISE_BUILD_NUMBER`, but this is not available locally, so we would need to increment a workflow env every time.

I suggest we make an exception and don't include a sample workflow in this step.